config/storage.yml

```diff
@@ -6,29 +6,17 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
-# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
+amazon:
+  service: S3
+  access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  region: 'us-west-2'
+  bucket: <%= ENV['AWS_BUCKET'] %>
+  
+# TODO: Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
```

Comment:
@veganstraightedge the rails docs suggest doing this, but i left the ENV var version above for now. lemme know if you want to change to this credentials api instead